sys/include/net/gnrc/netif/internal.h

```diff
@@ -748,4 +748,5 @@ int gnrc_netif_ipv6_add_prefix(gnrc_netif_t *netif,
 #endif
 
 #endif /* NET_GNRC_NETIF_INTERNAL_H */
-/** @} */
+/** @internal
+ *@} */
```

Comment:
Are you sure the `@internal` needs to be repeated at the end of group marker? IMO it should only be needed at the `@{`, not at the corresponding `@}`

Reply:
I think the group checker isn't working correctly (but i know if i will report that now it will reach riot in years (doxygen update -> ubuntu release  -> riot)